Modules/MUON/Common/include/MUONCommon/MergeableTH1Ratio.h

```diff
@@ -60,13 +60,15 @@ class MergeableTH1Ratio : public TH1F, public o2::mergers::MergeInterface
 
   void update();
 
+  void Reset(Option_t* option = "") override;
+
  private:
   TH1D* mHistoNum{ nullptr };
   TH1D* mHistoDen{ nullptr };
   std::string mTreatMeAs = "TH1F";
   double mScalingFactor = 1.;
 
-  ClassDefOverride(MergeableTH1Ratio, 1);
```

Comment:
thanks but actually in this instance it's not really needed as you're only adding methods, not data members.

Reply:
So should I revert the version?

Reply:
Actually, yes, please. That'll save me from changing the ccdb-proxy of mchview to deal with a new version ;-)

Reply:
I should actually remove the scaling factor parameter as I did for the MergeableTH2Ratio, in which case the bump in version would be justified.
However I can also keep the class as-is and revert the version bump if that's easier for you to handle.

On the other hand, is the version bump for  MergeableTH2Ratio ok? It was forgotten in the previous PR...

Reply:
If no data member are added/removed, I would keep the versions as they are. 
For TH2Ratio I had to "patch" the ccdb-proxy https://gitlab.cern.ch/alice-muon/ccdb-proxy/-/blob/master/ccdb/types.go#L154 and while it seems to work, I would like to avoid more patches like that if possible.

Reply:
Ok, so I revert both version bumps, right?

Reply:
yes, thanks.